### ISSUE TITLE
Remove 'pick_models' for randomly selecting SEDs from the physics model grid?

### ISSUE THREAD

**Issue by galaxyumi** [BEAST-Fitting/beast#504]
If I remember correctly, randomly selecting SEDs from the physics model grid to generate input ASTs was a relic from Heddy to try the truncheon method. Is there anyone use this functionality? If not, we can remove it or keep it as a place holder for a truncheon model. I personally think we can keep it for now and modify it later for the truncheon model.   

Any thoughts?

**Comment by karllark** [comment]
Humm....we won't want to randomly select fully for trunchen, but one idea does start include some random selection followed by the possibility of rejection.

**Comment by galaxyumi** [comment]
Yes, you are correct. If we keep it, it should be considered as a place holder. A small piece of it could be recycled as well?
